src/api/services/routing-private-auth-service.ts

```diff
@@ -13,18 +12,23 @@ export class RoutingPrivateAuthService implements CanActivate {
              private loginService: LoginService, private dotcmsConfig: DotcmsConfig) {}
 
     canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> {
-
-        return Observable.create( obs => {
+        return Observable.create(obs => {
             this.loginService.isLogin$.subscribe(isLogin => {
-
                 if (isLogin) {
-                    this.dotcmsConfig.getConfig().subscribe( configParams => {
-                        this.checkAccess(state.url).subscribe(checkAccess => {
-                            if (!checkAccess) {
-                                this.router.goToMain();
-                            }
-                            obs.next(checkAccess);
-                        });
+                    this.dotcmsConfig.getConfig().subscribe(configParams => {
+                        if (state.url.indexOf('home') > -1) {
+                            this.routingService.menusChange$.subscribe(res => {
+                                this.router.goToURL(`/c/${this.routingService.firstPortlet}`);
+                                obs.next(false);
```

Comment:
Try to do it, but somehow when you add routingService to dotroutingService explode!